vignettes/GenomicFiles.Rmd

```diff
@@ -0,0 +1,712 @@
+
+---
+title: "Introduction to GenomicFiles"
+author:
+  - Valerie Obenchain
+  - Michael Love
+  - Lori Shepherd
+  - Martin Morgan
+contributor:
+  - name: Sonali Kumari
+    affiliation: "Vignette translation from Sweave to R Markdown / HTML"
+date: "`r format(Sys.time(), '%B %d, %Y')`"
+package: GenomicFiles
+vignette: >
+  %\VignetteIndexEntry{Introduction to GenomicFiles}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document
+---
+
+# Introduction
+
+This vignette illustrates how to use the `GenomicFiles` package for distributed computing across files. The functions in `GenomicFiles` manipulate and combine data subsets via two user-supplied functions, `MAP` and `REDUCE`. These are similar in spirit to `Map` and `Reduce` in base R. Together they provide a flexible interface to extract, manipulate, and combine data. Both functions are executed in the distributed step, which means results are combined on a single worker, not across workers.
+
+We assume the reader has some previous experience with R and with basic manipulation of ranges objects such as `GRanges` and `GAlignments` and file classes such as `BamFile` and `BigWigFile`. See the vignettes and documentation in `GenomicRanges`, `GenomicAlignments`, `Rsamtools` and `rtracklayer` for an introduction to these classes.
+
+The `GenomicFiles` package is available at bioconductor.org and can be downloaded via `BiocManager::install`. 
+
+```
+> if (!require("BiocManager"))
++ install.packages("BiocManager")
+> BiocManager::install("GenomicFiles")
+```
```

Comment:
These should be R code blocks. (See Embed codes in https://www.rstudio.com/wp-content/uploads/2015/02/rmarkdown-cheatsheet.pdf). Typically, there is also a label so it can be identified with some attributes so that the code can be run (or not run). If you look at the corresponding .Rnw file, this part should be labeled `install` with `eval=FALSE` (https://github.com/Bioconductor/GenomicFiles/pull/8/files#diff-f1f895a7529de1d818ab65ace19caa98ab74c894fee01c0fe7060e57d30697c6L46-L50). It should look like

    ```{r install, eval=FALSE}
    if (!require("BiocManager"))
      install.packages("BiocManager")
    BiocManager::install("GenomicFiles")
    ```
    
I see all your code blocks don't have labels and they are not R code blocks but they should be. Please fix this code block and others in this file. Also take care of the spacing around the code blocks. There should be a blank line before and after a code block.

Reply:
Ohk .. I will do the changes and let you know after getting all the things done.. For now you can review other's codes